README.md

```diff
@@ -1,2 +1,48 @@
 # Technical-documentation-page
-creating new Technical-documentation-page.
+
+## Project Description 📝
+
+> give information about general coding.
+- for those people likes to learn coding.
+- give information about languge of coging like js, React... .
+
+  my project stand on html and css.
+
+```html
+- use the code like: p > this a paragraph element in HTML < /p >
+
+
+```css
+- use the code for styleling the page like: 
+p { text - align: is the CSS selector applied to align text of the paragraph element }
+
+
+
+## Technologies Used 🛠️
+
+- HTML
+- CSS
+
+## Installation 💻
+- until now it dose not need to install becase you just see the info in google.
+
+
+You can also get the code of this project in your locla machine by this command:
+- git clone "your ssh key"
+- git cd "name of your folder"
```

Comment:
- You have to copy and past your project ssh key in the "your ssh key" section
- And also your repo name 

And make it like a code example just by adding this:
  - Remove the parenthesis `(  )` and add your project ssh key link in the required section and also your project folder name:
```
(```
# git clone SSH KEY LINK OF YOUR OWN PROJECT HERE
```)
(```
# cd YOUR REPO NAME
```)
```